mayhem/Dockerfile

```diff
@@ -0,0 +1,45 @@
+ARG NAMESPACE=
```

Comment:
Ok. Have we tried using the build instructions from the README instead? Seems like it would run faster/require less dependencies.